salt/master.py

```diff
@@ -1909,6 +1910,14 @@ def publish(self, clear_load):
         payload = self._prep_pub(minions, jid, clear_load, extra, missing)
 
         # Send it!
+        if self.opts[u'enable_ssh_minions'] is True:
+            log.debug('Use SSHClient for rostered minions')
+            ssh = salt.client.ssh.client.SSHClient()
+            ssh_minions = ssh._prep_ssh(**clear_load).targets.keys()
+            if ssh_minions:
+                minions.extend(ssh_minions)
+                multiprocessing.Process(target=ssh.cmd, kwargs=clear_load).start()
```

Comment:
@s0undt3ch This process will go zombie if I am not mistaken. Can the process manager be modified to auto clean up a process when it dies instead of restarting it?

Reply:
Just checked, and it does become a zombie.